helmfile.d/values/networkpolicies/common/cert-manager.yaml.gotmpl

```diff
@@ -31,6 +31,15 @@ policies:
           ports:
             - tcp: 443
         {{- end }}
+        {{- with .Values | get "networkPolicies.certManager.dns01.ips" list }}
+        - name: dns01
+          peers:
+            {{- range . }}
+            - cidr: {{ . }}
+            {{- end }}
+          ports:
+            - tcp: 53
```

Comment:
Should we include udp here as well?

Reply:
Sure, I did not need it for route 53. But I do not see why not to add it.